src/EditorFeatures/CSharp/Formatting/CSharpEditorFormattingService.cs

```diff
@@ -39,7 +39,7 @@ public CSharpEditorFormattingService()
         public bool SupportsFormatDocument => true;
         public bool SupportsFormatOnPaste => true;
         public bool SupportsFormatSelection => true;
-        public bool SupportsFormatOnReturn => true;
+        public bool SupportsFormatOnReturn => false;
```

Comment:
All the changes in this file come from https://github.com/dotnet/roslyn/pull/33983  that PR should be merged first.